.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,7 +28,7 @@ Before creating a PR, run through this checklist and mark each as complete.
 
 ### Release notes
 
-If this PR introduces a change that affects users and needs to be mentioned in the [release notes](../CHANGELOG.md),
+If this PR introduces a change that affects users and needs to be mentioned in the [release notes](../blob/main/CHANGELOG.md),
```

Comment:
Does `/CHANGELOG.md` work?

Reply:
It does locally, but `../CHANGELOG.md` worked locally too...

Reply:
Does pushing the changes to your branch then opening a PR on your fork test if this works through the github site?

Reply:
I think `/CHANGELOG.md` should work. Typically, we try to avoid relative links, but it might be fine in this case cause I don't think the templates will be moved.

Reply:
`/CHANGELOG.md` redirects to `github.com/CHANGELOG.md` apparently.
I can use the full link if you think it's better.

Reply:
nah it's alright